Formula/csvtk.rb

```diff
@@ -25,6 +26,7 @@ class Csvtk < Formula
 
   def install
     system "go", "build", *std_go_args(ldflags: "-s -w"), "./csvtk"
+    generate_completions_from_executable(bin/"csvtk", "genautocomplete", shell_parameter_format: :arg)
```

Comment:
I don't think this will work because the command doesn't output the completions to stdout (and there's no way to get it to do so, I think). It writes the completions to a file by default and the only way to change it is by passing the `--file` argument.